torch/csrc/api/include/torch/nn/functional/instancenorm.h

```diff
@@ -0,0 +1,17 @@
+#pragma once
+
+namespace torch {
+namespace nn {
+namespace functional {
+
+inline Tensor instance_norm(const Tensor& input, const Tensor& running_mean,
+    const Tensor& running_var, const Tensor& weight, const Tensor& bias, 
+    bool use_input_stats, double momentum, double eps) {
```

Comment:
I think we can put the parameters
```python
running_mean=None, running_var=None, weight=None,
bias=None, use_input_stats=True, momentum=0.1, eps=1e-5
```
into `InstanceNormOptions` as well, so that the functional form can be:
```cpp
inline Tensor instance_norm(const Tensor& input, const InstanceNormOptions& options = {}) {
```